SDWebImage/SDDiskCache.m

```diff
@@ -260,9 +260,9 @@ - (void)moveCacheDirectoryFromPath:(nonnull NSString *)srcPath toPath:(nonnull N
         while ((file = [dirEnumerator nextObject])) {
             [self.fileManager moveItemAtPath:[srcPath stringByAppendingPathComponent:file] toPath:[dstPath stringByAppendingPathComponent:file] error:nil];
         }
-        // Remove the old path
-        [self.fileManager removeItemAtPath:srcPath error:nil];
     }
+    // Remove the old path
+    [self.fileManager removeItemAtPath:srcPath error:nil];
```

Comment:
What's the reason for moving this line?

Reply:
It's a bug or test code. Because when moving the old path `~/Library/Caches/default/com.hackemist.SDWebImageCache.default/` to new path `~/Library/Caches/com.hackemist.SDImageCache/default/`, I found that there are still a empty folder `~/Library/Caches/default/` there. So I remove it.

Actually this remove it's not safe, because `default` is a common name without any reverse domain, it's not good to remove it. I can just revert this line of code.